lib/gitsh/commands/internal_command.rb

```diff
@@ -104,7 +100,7 @@ def execute(env)
       private
 
       def valid_arguments?
-        args.length == 0 || args.length == 1
+        arg_values.length == 0 || arg_values.length == 1
```

Comment:
I'd normally agree with this Hound comment, but I disagree in this context because we're doing two size comparisons, and I think it's easier to read if they both use the same syntax.